### DISCUSSION TITLE
TypeError: Cannot destructure property 'GET' of '_src_lib_auth__WEBPACK_IMPORTED_MODULE_0__.handlers' as it is undefined.

### DISCUSSION THREAD

**Discussion by Rayane-HASSAINE** [vercel/next.js#66279] [Help]
### Summary

Hello i'm following a tutorial on how to make button authentification with github
i have this error i don't understand what to do

```
TypeError: Cannot destructure property 'GET' of '_src_lib_auth__WEBPACK_IMPORTED_MODULE_0__.handlers' as it is undefined.
app\api\auth\[...nextauth]\route.ts (2:15) @ GET

  1 |  import { handlers} from "@/src/lib/auth";
> 2 | export const {GET, POST} = handlers;
    |               ^
```

i pushed my code here so you have all the info https://github.com/Rayane-Truecolor/NextPrismaTuto

**Comment by Ronasunil** [comment]
Try this  if you are not planning to use beta version in auth.js
export const GET = NextAuth(authOptions);
export const POST = NextAuth(authOptions);

or at least install the newer version  of auth.js using  npm install next-auth@beta

The issue  is that you are using  auth.js version 5 but the installed package in json file would be 4 or something

NOTE: change import NextAuth from "next-auth/next";  to  import NextAuth from "next-auth";
